Dockerfile

```diff
@@ -16,4 +17,4 @@ ENV ELASTIC_LOG_PASSWORD='<ELASTIC-LOG-PASSWORD>'
 RUN apk update && apk add --no-cache bash
 RUN ls -ltra
 COPY gerrit ./
-CMD ./gerrit --gerrit-disable-host-key-check=true --gerrit-url=${GERRIT_URL} --gerrit-project=${GERRIT_PROJECT} --gerrit-es-url=${ES_URL} --gerrit-user=${GERRIT_USERNAME} --gerrit-ssh-key=${GERRIT_SSH_KEY}
+CMD ./gerrit --gerrit-disable-host-key-check=true --gerrit-url=${GERRIT_URL} --gerrit-project=${GERRIT_PROJECT} --gerrit-project-filter=${GERRIT_PROJECT_FILTER} --gerrit-es-url=${ES_URL} --gerrit-user=${GERRIT_USERNAME} --gerrit-ssh-key=${GERRIT_SSH_KEY}
```

Comment:
`--gerrit-project-filter` - it does not need to have a value, I mean it is not mandatory. correct?
If this is the case, then the below env variables need not be set, I think.